generate_default_value_set_list.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import json
+import logging
+import os
+import sys
+
+import deepl
+
+logger = logging.getLogger(__name__)
+
+def configure_args_parser():
+    arg_parser = argparse.ArgumentParser(description="Generate Translations for fdpg terms")
+    arg_parser.add_argument("--value_sets_folder", type=str, help="Folder with value-sets to be listed in json",
+                            nargs="?", default="value-sets")
+    arg_parser.add_argument("--log_level", type=str, help="notset|debug|info|warning|error|critical", nargs="?",
+                            default="info")
+    arg_parser.add_argument("--deepl_api_key", type=str, help="deepl api key", nargs="?",default="no_key")
+    arg_parser.add_argument("--lang_detection", action='store_true', help="without language detection")
+
+    return arg_parser
+
+
+def configure_logging(log_level_arg):
+    log_level = {
+        "notset": 0,
+        "debug": 10,
+        "info": 20,
+        "warning": 30,
+        "error": 40,
+        "critical": 50
+    }
+    logging.basicConfig(
+        stream=sys.stdout,
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        level=log_level[log_level_arg],
+    )
+    logging.getLogger("deepl").disabled = True
+
+
+if __name__ == "__main__":
+    args = configure_args_parser().parse_args()
+    configure_logging(args.log_level)
+
+    deepl_engine = None
+    if args.deepl_api_key != "no_key":
```

Comment:
@Frontman50 i would do this differently - check if --lang_detection == true and if so and no api key return with an error -